### ISSUE TITLE
options for svg badge

### ISSUE THREAD

**Issue by r3mi** [rvagg/nodei.co#16]
It seems it is possible to get a SVG badge by replacing the .png extension in the URL with .svg , e.g. https://nodei.co/npm/node.svg
However, it works only for the default badge : adding options reverts to a PNG image, e.g. https://nodei.co/npm/node.svg?compact=true.
Is it the intended behavior ? Or is SVG unsupported, given it is not described on the homepage ?

**Comment by yuchi** [comment]
Are you still interested in this?

**Comment by rvagg** [comment]
yes, got some time to spare tinkering with this?

**Comment by yuchi** [comment]
Yes and no, but I’d like to see all those badges in hi-res and looks like an easy enough task to contribute back to this great project.

**Comment by GarthDB** [comment]
Another thing to tack on to the svg support - sending the correct headers. Right now the `content-type` is `text/html`

```sh
$ curl -L -I https://nodei.co/npm/topmarks.svg
HTTP/1.1 200 OK
x-startup: Tue Jul 12 2016 02:37:01 GMT+0000 (UTC)
x-powered-by: whatevs
cache-control: no-cache
Content-Type: text/html
Content-Length: 2573
Date: Sun, 17 Jul 2016 17:12:26 GMT
Strict-Transport-Security: max-age=31536000; includeSubDomains
```

It should be set to `image/svg+xml`

**Comment by GarthDB** [comment]
I can open it as a new issue if you'd prefer.
